src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4600,6 +4797,16 @@ namespace VariableSpeedCoils {
 		SourceSideInletEnth = Node(CondInletNode).Enthalpy;
 		VarSpeedCoil(DXCoilNum).InletWaterEnthalpy = SourceSideInletEnth;
 
+		//Check for flows, do not perform simulation if no flow in load side or source side.
+		if ((SourceSideMassFlowRate <= 0.0) || (LoadSideMassFlowRate <= 0.0)) {
+			VarSpeedCoil(DXCoilNum).SimFlag = false;
+			return;
```

Comment:
There wasn't an early `return` here before.  Is there anything that needs to be cleaned up before exiting this function?